.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ workflows:
       - orb-tools/review:
           filters: *filters
       - shellcheck/check:
-          exclude: SC2148,SC2038,SC2086,SC2002,SC2016,SC1090
+          exclude: SC2148,SC2038,SC2086,SC2002,SC2016,SC1090,SC2129
```

Comment:
I know that we used `SC2129` syntax here and I do not believe it should be added to the exclude list.

Reply:
Removed SC2129